css/common.css

```diff
@@ -21,15 +21,26 @@ body {
   );
   color: #eeeeee;
 }
+
 #loader{
   position: fixed;
   top: 0;
   left: 0;
   width: 100vw;
   height: 100vh;
-  background-color: #fff;
   z-index: 99999;
+
+  background: linear-gradient(230deg, #180834, #26032c, #101010);
+  background-size: 600% 600%;
+  animation: loader-bg 3s ease infinite;
 }
+
+@keyframes loader-bg {
+  0%{background-position:50% 0%}
+  50%{background-position:50% 100%}
+  100%{background-position:50% 0%}
+}
+
 header {
```

Comment:
Looks good. In principle, this CSS file may itself become very large (not sure how large it currently is) and so what you really want to do is to first load *just* the CSS required for the loader, then load the loader components, and **then** load the rest of the CSS. (this is not an issue for Js and HTML because those anyway load after the loader).